src/components/Scrollable/Scrollable.tsx

```diff
@@ -112,6 +114,7 @@ class Scrollable extends React.Component<CombinedProps, State> {
       vertical = true,
       shadow,
       hint,
+      onScrollToBottom: _unused,
```

Comment:
Do we need this?

Reply:
https://github.com/Shopify/polaris-react/blob/e4be478abc715cb599eabba3f6dfb87f693b8798/src/components/Scrollable/Scrollable.tsx#L134

Will spread the `rest` on the `div`. But I didn't want to have `onScrollToBottom` added to the `div` that why I removed it here. Does it make sense?

Reply:
I would rather clean up the rest spread on the div. Not a fan of that rest spread to begin with.

Reply:
Was this added to TS recently? I remember having some issues a while ago. 
Removed `_unused`.

Reply:
I'm not sure, when I first used it a couple months ago it worked. But have have been different before that